app/pricelist.js

```diff
@@ -28,96 +28,107 @@ pricelist.addItems = async function(search, options) {
 		}
 	}
 
-	return getAllPrices()
-		.then(async (prices) => {
-			console.log('Got all prices, continuing...');
-
-			for (let i = 0; i < prices.length; i++) {
-				if (search.indexOf(prices[i].name) > -1) {
-					skus.push(prices[i].sku);
-					search.splice(search.indexOf(prices[i].name), 1);
-				}
+	if (options.autoprice) {
+		const prices = await getAllPrices();
```

Comment:
Should maybe catch an error here

Reply:
Errors wil get caught in addItems.js, no need to rethrow them.